src/System.DirectoryServices/tests/System/DirectoryServices/ActiveDirectory/ForestTests.cs

```diff
@@ -53,15 +53,25 @@ public void GetForest_NonNullNameAndNotRootedDomain_ThrowsActiveDirectoryObjectN
         [InlineData(DirectoryContextType.Forest, "\0")]
         [InlineData(DirectoryContextType.DirectoryServer, "server:port")]
         [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Not approved COM object for app")]
-        public void GetForest_NonNullNameAndNotRootedDomain_ThrowsActiveDirectoryObjectNotFoundException_NonUap(DirectoryContextType type, string name)
+        public void GetForest_NonNullNameAndNotRootedDomain_NonUap(DirectoryContextType type, string name)
         {
             var context = new DirectoryContext(type, name);
             if (!PlatformDetection.IsDomainJoinedMachine)
             {
                 Assert.Throws<ActiveDirectoryObjectNotFoundException>(() => Forest.GetForest(context));
```

Comment:
Did you forget to remove the 1st assert? I expect it will still fail (in nightly runs) ...

Reply:
Thanks for the catch. I have submitted the PR https://github.com/dotnet/corefx/pull/26555